lambda_function/requirements.txt

```diff
@@ -1 +1,6 @@
-psycopg2-binary==2.8.6
\ No newline at end of file
+<<<<<<< HEAD
```

Comment:
these changes should be reverted, only lambda libs should be added here

Reply:
resolved